tests/test_top.py

```diff
@@ -9,6 +9,454 @@
 from utils.sram_macro import SRAMMacroInfo
 
 
+def test_mult_lines_dim1(data_width=16,
```

Comment:
Can these two tests be merged into a parametrized test? It seems that these are testing the same feature, so it would be clean to merge.